### PR TITLE
Ana/make backtovalidators button smart

### PR DESCRIPTION
**Pull request by Bitcoinera**
@jbibla this one is for you!

It is not as perfect as I would have wished, since it seems there is no straightforward way to capture the last visited page. But I think just changing "Back to Validators" for "Back to previous page" is good enough, I checked responsiveness with the button and looks good.

Now the back button does go back to any page you were before within the app.

Thank you! 🚀

EDIT: OK, I acknowledge it is a bit hacky

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
We wanted to scrap the button all together. @jbibla what is your thought?

**Comment by Bitcoinera** [comment]
> We wanted to scrap the button all together. @jbibla what is your thought?

I will try today using Lunie on a Tablet, but I'm afraid that the three dots thing is not so UXish.

We should think on how to make Lunie more navigable imo

**Comment by faboweb** [comment]
Tablet should probably have the same navigation as mobile in portrait and desktop in landscape mode

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -12,14 +12,10 @@
...
-          @click="
-            $router.push(
-              `/${$router.history.current.params.networkId}/validators`
-            )
-          "
+          @click="$router.go(-1)"
         >
           <i class="material-icons notranslate arrow">arrow_back</i>
-          Back to Validators
+          Back to previous page
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue:18]:
```suggestion
          Back
```

**Comment by jbibla** [comment]
this is a good improvement for now. one day i hope we don't need these buttons.

**Pull request merged**